src/trulens_eval/trulens_eval/database/migrations/__init__.py

```diff
@@ -0,0 +1,15 @@
+# ruff: noqa: E402, F403, F401
+"""
+!!! warning
+    This module is deprecated and will be removed. Use
+    `trulens.core.database.migrations` instead.
+"""
+
+from trulens.core.utils import deprecation as deprecation_utils
+
+deprecation_utils.packages_dep_warn("trulens_eval.database.migrations")
+
+# NOTE: This file had contents in trulens_eval none were public or aliases.
+# Because of that, this backwards compatibility module is empty.
+
+from trulens.core.database.migrations import *
```

Comment:
I'm confused what's going on here, in lines 12-13 you said it's empty, so what are you importing here?

Reply:
I think it was just a copy and paste to capture `__all__` exports but didn't remove it after realizing there was no `__all__`. However, I'm leaving this file in place because I want to also make the non `__all__` names also backwards compatible in the next PR.

Reply:
* I'm deleting the `import *`, but leaving the file in place is what I meant.

Reply:
Also adding a note explaining this.